gamemaster/game.php

```diff
@@ -1050,6 +1051,33 @@ protected function setWon(Member $Winner)
 		 */
 		global $DB;
 		$DB->sql_put("UPDATE wD_Games SET finishTime=".time()." WHERE id=".$this->id);
+		
+		//Do GR Stuff

```

Comment:
Should all of the logic around GR be wrapped in a check that sees if the config section for GR exists? That way if a dev instance doesn't have it setup or another site chooses not to use it there's no issue?

Reply:
Good idea, and shouldn't be a problem to implement. I'll add this in my next commit.

Reply:
Just put in a commit which addresses this.